cmd/svcat/broker/register_cmd_test.go

```diff
@@ -111,7 +111,7 @@ var _ = Describe("Register Command", func() {
 			}
 			err := cmd.Validate([]string{"bananabroker", "http://bananabroker.com", "--basic-secret", basicSecret, "--bearer-secret", bearerSecret})
 			Expect(err).To(HaveOccurred())
-			Expect(err.Error()).To(ContainSubstring("cannot use both basic auth and bearer auth"))
```

Comment:
would be nice to have some actual tests for the new stuff added - the new flags, the added case(s) in validate(), and the new secret generation in run() with the pass-off of the new secrets name in opts

Reply:
Sure I'll add them